backend/dataall/base/utils/api_handler_utils.py

```diff
@@ -24,13 +24,22 @@
 ]
 ENGINE = get_engine(envname=ENVNAME)
 ALLOWED_ORIGINS = os.getenv('ALLOWED_ORIGINS', '*')
+AWS_REGION = os.getenv('AWS_REGION')
 
 
 def redact_creds(event):
-    if 'headers' in event and 'Authorization' in event['headers']:
+    if event.get('headers', {}).get('Authorization'):
         event['headers']['Authorization'] = 'XXXXXXXXXXXX'
-    if 'multiValueHeaders' in event and 'Authorization' in event['multiValueHeaders']:
+
+    if event.get('multiValueHeaders', {}).get('Authorization'):
         event['multiValueHeaders']['Authorization'] = 'XXXXXXXXXXXX'
+
+    if event.get('multiValueHeaders', {}).get('accesskeyid'):
+        event['multiValueHeaders']['accesskeyid'] = 'XXXXXXXXXXXX'
+
+    if event.get('headers', {}).get('accesskeyid'):
+        event['headers']['accesskeyid'] = 'XXXXXXXXXXXX'
+
```

Comment:
in `deploy/custom_resources/custom_authorizer/custom_authorizer_lambda.py` the only values that we actually use are coming from the headers. Is it possible that the API succeeds with multiVValueHeaders? and if not, maybe we don't need these checks here. (It might be out of scope for this PR)
```
auth_token = incoming_event['headers']['Authorization']
access_token = incoming_event['headers']['accesskeyid']
```